cmd/argo/commands/common.go

```diff
@@ -34,6 +34,7 @@ var (
 	// DEPRECATED
 	wftmplClient     v1alpha1.WorkflowTemplateInterface
 	jobStatusIconMap map[wfv1.NodePhase]string
+	nodeTypeIconMap  map[wfv1.NodeType]string
```

Comment:
Is there a reason you added a new map for this? I think this could go in `jobStatusIconMap`

Reply:
Basically two reasons for this:
- `Suspend` is a `NodeType` rather than a status; adding this symbol to `jobStatusIconMap` sounds incorrect
- There's no union type in golang, so I cannot create a new entry in `jobStatusIconMap` with  `NodeType` unless we create something like an interface, but what should we call this interface?

Reply:
:+1: